exec/plugin-commands-script-runners/src/dlx.ts

```diff
@@ -91,7 +91,19 @@ export async function handler (
       saveOptional: false,
       savePeer: false,
     }, pkgs)
-    await symlinkDir(cachedDir, cacheLink, { overwrite: true })
+    try {
+      await symlinkDir(cachedDir, cacheLink, { overwrite: true })
+    } catch (error) {
+      // EBUSY means that there is another dlx process running in parallel that has acquired the cache link first.
+      // The link created by the other process is just as up-to-date as the link the current process was attempting
+      // to create. Therefore, instead of re-attempting to create the current link again, it is just as good to let
+      // the other link stay. The current process should yield.
+      if (util.types.isNativeError(error) && 'code' in error && error.code === 'EBUSY') {
+        await new Promise(resolve => setTimeout(resolve, 0))
```

Comment:
is this line needed? Isn't the link already available?

Reply:
Initially I thought I should wait for the other process to finish creating the symlink, but after I found that `cacheLink` isn't used by the remaining steps, I think it is safe to remove it: https://github.com/pnpm/pnpm/pull/8604/commits/18558df2780be07d31cf7858fe3a63b79aaafbef